src/NavBar/NavBar.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import {StyleSheet, Text, View} from 'react-native';
+import {NativeRouter, Route, Link} from 'react-router-native';
+
+const NavBar = ({match}) => (
+  <View style={styles.container}>
+    <View style={styles.nav}>
+      <Link to="/about" underlayColor="#f0f4f7" style={styles.navItem}>
+        <Text style={styles.topic}>About</Text>
+      </Link>
+      <Link to="/" underlayColor="#f0f4f7" style={styles.navItem}>
```

Comment:
Like previously mentioned, we may have to adjust this route to be `/categories` so that we can make our welcome/landing page to be the `/` route.